src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+  display: flex;
+  list-style: none;
+  margin: 0;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
make it in the reset.scss